src/Alg/Signature/SignatureAlgorithmFactory.php

```diff
@@ -38,44 +36,37 @@ final class SignatureAlgorithmFactory
      *
      * @var string[]
      */
-    private array $blacklist = [
+    protected array $blacklist = [
```

Comment:
same

Reply:
Same reason. If private, the abstract class won't be able to access it.